clusterloader2/testing/density/config.yaml

```diff
@@ -3,6 +3,7 @@
 # - Number of nodes should be divisible by NODES_PER_NAMESPACE (default 100).
 
 #Constants
+{{$DENSITY_RESOURCE_CONSTRAINTS_FILE := DefaultParam .DENSITY_RESOURCE_CONSTRAINTS_FILE ""}}
```

Comment:
I'm not sure I understand this.
So the goal is to set two different files, the second file setting the override?

How this will look like on test-infra side?

Reply:
All overrides are provided via yaml file. On test-infrs side it will require additional test argument
```
--testoverrides=./testing/density/100_nodes/override.yaml
```

Reply:
got it - thanks